internal/cmd/kafka/command_broker_delete.go

```diff
@@ -17,7 +17,7 @@ import (
 
 func (c *brokerCommand) newDeleteCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "delete <id>",
+		Use:   "delete <broker-id>",
```

Comment:
The existing text is less redundant, what do you think?

Reply:
Same as above: I'm matching the list output.

Reply:
> I'm matching the list output.

What do you mean? The column name of `confluent kafka broker list`?

Reply:
Yeah.

Reply:
Seems a bit unrelated. The use case here is a user wants to delete a broker, so they type `confluent kafka broker delete -h`. Just a simple `<id>` seems sufficient since they've already typed "broker" and they know what resource they're about to delete.

Reply:
Alright. I'll make the change, but I think we should change the field to ID in the get outputs in v4 for the same reason.

Reply:
Feel free to make a ticket!

Reply:
Made this change for `partition-id` as well.